### ISSUE TITLE
Please release a new gem.

### ISSUE THREAD

**Issue by damncabbage** [cehoffman/sinatra-respond_to#16]
I'm using Sinatra::RespondTo, and just got bitten by the _"Call 'register Sinatra::RespondTo' twice and have all requests call the :html format block"_ bug.

In the interests of saving the time of future pit-stumbler-into-ers, please publish a new version of the gem.

Until then, I recommend people use the following in their Gemfiles:

```
gem 'sinatra-respond_to', :require => 'sinatra/respond_to',
                          :git => 'git://github.com/cehoffman/sinatra-respond_to.git',
                          :ref => '0b8892eb188fb7ba9d1'
```

**Comment by cehoffman** [comment]
I've pushed out a new version that includes the accept header usage. However, the reason you saw the referenced issue is the before filter that lets respond to do its work wasn't meant to be run more than once. The additional code that gkellogg added still has this assumption; it just happens to work.

I've looked at adding a guard on the register call to detect if the app already has RespondTo registered, but there isn't a good way to do it because sinatra adds RespondTo to the list of extensions on the app before it actually calls the registered callback.

I suggest a guard on the register calls similar to `app.register Sinatra::RespondTo unless app.extensions.include? Sinatra::RespondTo`.
